resources/public/pages-output/about/index.html

```diff
@@ -67,7 +67,7 @@
         <h2>About</h2>
     </div>
     
-    <p>The Bangalore Clojure Group hosts one of the most active (since 2013) meetups for Clojure in India, each of our meetups are conducted in different locations with different set of speakers every month.Our meetups are scheduled conveniently on every first Saturday of the month.</p><p>Our meetups usually have talks by clojurists, but we also do organise hackathons on a regular basis. We are an open community, we encourage all Clojurists & functional freaks without any bais (We also welcome windows users 🙂)</p><p>Happy Clojuring !</p><p>Some of our venue sponsors are:</p><ul><li>SAP Concur</li><li>Quintype</li><li>Nilenso</li><li>Go-Jek</li></ul>
+    <p>The Bangalore Clojure Group hosts one of the most active (since 2013) Clojure meetups in India. Each of our meetups are conducted at different locations with different set of speakers every month. Our meetups are scheduled conveniently on every first Saturday of the month.</p><p>Our meetups usually have talks by clojurists, but we organise hackathons on regular basis as well. We are an open community, we encourage all Clojurists & functional freaks without any bais (We also welcome windows users 🙂)</p><p>Please reach out to us if you have any thoughts on what we can do together or how we can help one another.</p><p>Ways to reach us:</p><ul><li><a href='https://clojurians.herokuapp.com/'>Clojurians slack</a>: <code>#bangalore-clj</code> channel</li><li><a href='https://groups.google.com/forum/#!forum/bangalore-clj'>Google group</a></li><li>Email: <code>bangaloreclojure@gmail.com</code></li></ul><p>Happy Clojuring !</p><p>Some of our meetup venue sponsors are:</p><ul><li>SAP Concur</li><li>Quintype</li><li>Nilenso</li><li>Go-Jek</li></ul>
```

Comment:
The meetups started much earlier than 2013 - check https://groups.google.com/forum/#!forum/bangalore-clj older posts for details.

Reply:
fixed